app/commands/track/set_file_system_permissions.rb

```diff
@@ -0,0 +1,21 @@
+class Track::SetFileSystemPermissions
+  include Mandate
+
+  initialize_with :track
+
+  def call
```

Comment:
Could we add a guard clause to this to check if a track is already safe? Just thinking of any future infinitely recursive deploys we could end up in if something goes wrong.

Reply:
Good idea. I've added the check